docs/javascript/extra.js

```diff
@@ -1,91 +0,0 @@
-const Amplify = window.aws_amplify.Amplify
-const Analytics = Amplify.Analytics
-const KinesisFirehoseProvider = window.aws_amplify.AWSKinesisFirehoseProvider
-
-const awsconfig = {
-	"aws_project_region": "eu-west-1",
-	"aws_cognito_identity_pool_id": "eu-west-1:3df3caec-4bb6-4891-b154-ee940c8264b8",
-	"aws_cognito_region": "eu-west-1",
-	"aws_kinesis_firehose_stream_name": "ClickStreamKinesisFirehose-OGX7PQdrynUo",
-};
-
-const RUNTIME = "dotnet"
-const BASE_ORIGIN = "docs.powertools.aws.dev"
-
-function copyToClipboard(e) {
-	e.preventDefault()
-	navigator.clipboard.writeText(e.target.textContent)
-	alert$.next("Copied to clipboard")
-}
```

Comment:
You need to keep this as well as `enableClipboardElements`, `attachListeners`, and call it at the end of the file like `attachListeners()` (where now `init()` is).

Reply:
done